pkgs/applications/gis/qgis/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
   # To handle the lack of 'local' RPATH; required, as they call one of
   # their built binaries requiring their libs, in the build process.
   preBuild = ''
-    export LD_LIBRARY_PATH=`pwd`/output/lib:$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH=`pwd`/output/lib:${stdenv.lib.makeLibraryPath [ openssl ]}:$LD_LIBRARY_PATH
```

Comment:
Pretty sure the default setup does this for you; simply adding `openssl` to build inputs should be enough. If you intend to specify this both here and below, please factor out a `libPath` attribute to ensure that they remain in sync.